DailyReportWeb/Controllers/ReportController.cs

```diff
@@ -5,20 +5,28 @@
 using System.Web;
 using System.Web.Mvc;
 using DailyReportWeb.Helpers;
+using SourceControlLogReporter.Model;
+using System.Configuration;
 
 namespace DailyReportWeb.Controllers
 {
     public class ReportController : Controller
     {
+
         [HttpGet]
         public ActionResult Send(string id, DateTime date)
         {
             if (date.Date != DateTime.Today)
                 return Content("Cannot confirm report for another date");
 
-            if (ReportRunner.TryRunReport(id))
+            var policy = PolicyService.LoadPolicy(id);
+            var policyPath = PolicyService.GetPolicyPath(id);
+
+            PolicyService.SetPolicyFinalReport(policy, policyPath);
```

Comment:
@DanBadea 
Pai verificarea se face de fiecare data cand se confirma un raport individual. Se verifica daca toate drafturile individuale s-au confirmat, si apoi se trimite draftul final (in cazul in care toate drafturile individuale sunt confirmate). Iar dupa ce se trimtie draftul final, se confirma. Actiunea asta e pusa le linkul de confirmare, deci draftul a fost confirmat si se trimtie raportul final.